src/libcommonserver/utility/utility.cpp

```diff
@@ -273,13 +273,13 @@ std::string Utility::formatGenericServerError(std::istream &inputStream, const P
     return errorStream.str();   // str() return a copy of the underlying string
 }
 
-void Utility::logGenericServerError(const std::string &errorTitle, std::istream &inputStream,
+void Utility::logGenericServerError(const log4cplus::Logger &logger, const std::string &errorTitle, std::istream &inputStream,
                                     const Poco::Net::HTTPResponse &httpResponse) {
     std::string errorMsg = formatGenericServerError(inputStream, httpResponse);
 #ifdef NDEBUG
     sentry_capture_event(sentry_value_new_message_event(SENTRY_LEVEL_WARNING, errorTitle.c_str(), errorMsg.c_str()));
 #endif
-    LOG_WARN(_logger, errorTitle.c_str() << ": " << errorMsg.c_str());
```

Comment:
What to do with all the other uses of `Utility::_logger`? It seems to relate to https://infomaniak.atlassian.net/browse/KDESKTOP-738.

Reply:
I did not see any crash related to other call to logger. Do you think we should apply the same strategy everywhere in this Utility file?

Reply:
I cannot tell without further investigation. I just wanted to know if you have clear understanding of the reason of this specific crash and if this fix can or should apply at other places. We can certainly wait and address this with https://infomaniak.atlassian.net/browse/KDESKTOP-738.

Reply:
I definitly agree that we should address https://infomaniak.atlassian.net/browse/KDESKTOP-738, perhaps we should discuss the priority of this issue in next meeting.
Since this fix will be released in the next version, I prefer keeping the changes to a minimum. I will open other issues if I detect other crashes